holoviews/core/util.py

```diff
@@ -1263,6 +1263,13 @@ def get_param_values(data):
     return params
 
 
+def is_param_method(obj):
```

Comment:
Seems like something that could live in param...

Reply:
Agreed, I'll add that in a PR, I'm also duplicating it in ``panel``, so that would definitely make sense.